totalRP3/Modules/Register/Main/RegisterTooltip.lua

```diff
@@ -84,6 +84,17 @@ local ConfigKeys = {
 	PETS_INFO = "tooltip_pets_info",
 };
 
+TRP3_TooltipCroppingConstants = {
+	Title = 100,
+	Name = 50,
+	Race = 30,
+	Class = 30,
+	Pronouns = 30,
+	GuildName = 30,
+	GuildRank = 30,
+	VoiceReference = 30,
+}
```

Comment:
```suggestion
};
```

WHERE'S THE LAMB SAUCE